File: Estructura_de_datos_Python/burbuja.py
import random
import os
MINIMO = 1
MAXIMO = 10
array = [0]*10
for i in range(len(array)):
    array[i] = random.randrange(MINIMO, MAXIMO)

##antes de ordenar
print("Array antes de ordenar")
print(array)
##ordenamiento burbuja
for i in range(len(array)):
    for j in range(0, len(array)-i-1):
        if array[j] > array[j+1]:
            array[j], array[j+1] = array[j+1], array[j]
##despues de ordenar
print("Array despues de ordenar")
print(array)

File: Ordenamiento_C++/Burbuja.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;
int main(){
    int array[10];
    int MINIMO = 1;
    int MAXIMO = 10;
    int auxiliar;
    srand(time(NULL));
    for (int i = 0; i < 10; i++){
        array[i] = rand() % (MAXIMO - MINIMO + 1) + MINIMO;
    }
    // mostrar antes de ordenar
    cout << "Antes de ordenar: ";
    for (int i = 0; i < 10; i++){
        cout << array[i] << " ";
    }
    cout << endl;
    //ordenar con metodo burbuja
    for (int i = 0; i < 10 ; i++){
        for (int j = 0; j < 10 - i -1; j++){
            if (array[j] > array[j + 1 ]){
                auxiliar = array[j];
                array[j] = array [j+1];
                array[j+1] = auxiliar;
            }
        }

    }
    // mostrar despues de ordenar
    cout << "Despues de ordenar: ";
    for (int i = 0; i < 10; i++)
        cout << array[i] << " ";

return 0;
}
